### PR TITLE
App page navigation

### PR DESCRIPTION
**Pull request by davidfateh**
# Purpose of PR
Adding in a new method to allow for navigation to app page locations.

## PR Checklist

- [x] Tests are added/updated/not required
- [x] Tests are passing
- [x] Typescript typings are added/updated/not required

### CODE REVIEW

File: lib/navigator.js [hunk 1]
```diff
@@ -32,7 +32,10 @@ module.exports = function createNavigator(channel, currentExtensionId) {
       return channel.call('navigateToContentEntity', { ...opts, entityType: 'Asset', id: null })
     },
     openPageExtension: opts => {
-      return channel.call('navigateToPageExtension', { id: currentExtensionId, ...opts })
+      return channel.call('navigateToPage', { type: 'extension', id: ids.extension, ...opts })
+    },
+    openCurrentAppPage: opts => {
+      return channel.call('navigateToPage', { type: 'app', id: ids.app, ...opts })
```
Comment (jelz) [inline lib/navigator.js:38]: I'd put `id` after `opts` so it's not possible to override

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "contentful-ui-extensions-sdk",
   "description": "SDK to develop custom UI Extension for the Contentful Web App",
-  "version": "3.12.1",
+  "version": "3.13.0",
```
Comment (jelz) [inline package.json:4]: I'd have one more small thing before wee do 3.13 - can we do it later?

Comment (jelz) [reply package.json:4]: Also let's wait for #284 - we should be merging tomorrow. cc @taye

File: lib/navigator.js (hunk 1 above)
Comment (davidfateh) [reply lib/navigator.js:38]: I did this purposely so it is possible to override. Right now the web app won't accept the override but in the future we may want to accept certain overrides.

**Pull request merged**